src/components/ApplicationList/Applications/Application.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  IApplicationsListProps,
+  IDeactivateApplicationProps,
+} from '@allTypes/reduxTypes/areaSpecializationTypes'
+import { Roles } from '@allTypes/reduxTypes/usersStateTypes'
+import { ApplicationMenu } from '@components/ApplicationList/ApplicationMenu'
+import { CloneIcon } from '@components/Icons/CloneIcon'
+import { DeactivateIcon } from '@components/Icons/DeactivateIcon'
+import { EditIcon } from '@components/Icons/EditIcon'
+import { Translation } from '@constants/translations'
+import { dispatch, useAppSelector } from '@redux/hooks'
+import { applicationsMiddleware } from '@redux/slices/applications'
+import { usersSelector } from '@redux/slices/users'
+import { viewsMiddleware } from '@redux/slices/views'
+import { Button } from '@uiComponents/Button'
+
+interface IApplicationsProps {
+  item: IApplicationsListProps
+  notificationId: string | null
+}
+
+export const Application = ({ item, notificationId }: IApplicationsProps) => {
```

Comment:
```suggestion
export const Application = ({ isActive, id, notificationId }: IApplicationsProps) => {
```

Reply:
we must send item because except isActive and id we use from item another keys